File: Kapsch.Core.Gateway.Models/MobileDevice/MobileDeviceApplicationModel.cs
using Kapsch.Core.Gateway.Models.Enums;
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Kapsch.Core.Gateway.Models.MobileDevice
{
   
    public class MobileDeviceApplicationModel
    {
        public long ID { get; set; }
        public string Name { get; set; }
        public string SoftwareVersion { get; set; }
        public ApplicationType ApplicationType { get; set; }
        public Status Status { get; set; }
        public DateTime? ModifiedTimestamp { get; set; }       
    }
}

File: Kapsch.MobileApps/KapschMobile/console/src/main/java/za/co/kapsch/console/Models/MobileDeviceApplicationModel.java
package za.co.kapsch.console.Models;

import com.google.gson.annotations.SerializedName;
import com.j256.ormlite.field.DatabaseField;
import com.j256.ormlite.table.DatabaseTable;

import java.util.Date;

import za.co.kapsch.console.Enums.ApplicationStatus;
import za.co.kapsch.console.Enums.ApplicationType;

/**
 * Created by CSenekal on 2017/07/20.
 */

@DatabaseTable(tableName = "MobileDeviceApplication")
public class MobileDeviceApplicationModel {

    @DatabaseField(columnName = "ID", id = true)
    @SerializedName("ID")
    public long mID;

    @DatabaseField(columnName = "Name")
    @SerializedName("Name")
    public String mName;

    @DatabaseField(columnName = "SoftwareVersion")
    @SerializedName("SoftwareVersion")
    public String mSoftwareVersion;

    @DatabaseField(columnName = "ApplicationType")
    @SerializedName("ApplicationType")
    public ApplicationType mApplicationType;

    @DatabaseField(columnName = "Status")
    @SerializedName("Status")
    public ApplicationStatus mStatus;

    @DatabaseField(columnName = "ModifiedTimestamp")
    @SerializedName("ModifiedTimestamp")
    public Date mModifiedTimestamp;

    public long getID() {
        return mID;
    }

    public void setID(long ID) {
        mID = ID;
    }

    public String getName() {
        return mName;
    }

    public String getSoftwareVersion() {
        return mSoftwareVersion;
    }

    public ApplicationType getApplicationType() {
        return mApplicationType;
    }

    public ApplicationStatus getStatus() {
        return mStatus;
    }

    public Date getModifiedTimestamp() {
        return mModifiedTimestamp;
    }
}
